cmd/lotus-storage-miner/init_restore.go

```diff
@@ -30,250 +31,259 @@ import (
 	"github.com/filecoin-project/lotus/node/repo"
 )
 
-var initRestoreCmd = &cli.Command{
-	Name:  "restore",
-	Usage: "Initialize a lotus miner repo from a backup",
-	Flags: []cli.Flag{
-		&cli.BoolFlag{
-			Name:  "nosync",
-			Usage: "don't check full-node sync status",
-		},
-		&cli.StringFlag{
-			Name:  "config",
-			Usage: "config file (config.toml)",
-		},
-		&cli.StringFlag{
-			Name:  "storage-config",
-			Usage: "storage paths config (storage.json)",
-		},
-	},
-	ArgsUsage: "[backupFile]",
-	Action: func(cctx *cli.Context) error {
-		log.Info("Initializing lotus miner using a backup")
-		if cctx.Args().Len() != 1 {
-			return xerrors.Errorf("expected 1 argument")
-		}
+func restore(ctx context.Context, cctx *cli.Context, manageConfig func(*config.StorageMiner) error, after func(api lapi.FullNode, addr address.Address, peerid peer.ID, mi miner.MinerInfo) error) error {
```

Comment:
FYI, I wasn't able to review this file because the diff is too noisy and I'm not familiar with this command. I trust @magik6k here.